modules/ROOT/pages/jdbc-tracing.adoc

```diff
@@ -192,12 +193,51 @@ The Oracle driver supports the `java.util.logging` library. The driver uses the
 </jdbcDriver>
 ----
 
-The trace that is produced by the debugging driver is limited. You can add the following option to the `jvm.options` file to ensure that all trace is produced.
+Add the following option to the `jvm.options` file to ensure that all trace is produced.
 [source, txt]
 ----
 -Doracle.jdbc.Trace=true
 ----
 
+[#OracleCustom]
+==== Oracle customized tracing
+
+The Oracle driver tracing will be combined with Liberty tracing by default since both utilize the `java.util.logging` library. As of version 23.0.0.2, Liberty has added support for the seperation of Oracle trace from Liberty trace. This will allow users to provide standalone Oracle JDBC trace when requested by Oracle support.
+
+The following `server.xml` file example shows you how to configure the Oracle debugging driver.  No trace specification is required.
+
+[source, xml]
+---
+<jdbcDriver id="oracleDriver">
+  <library id="oracleDebug">
+      <file name="path_to_oracle_driver/ojdbcX_g.jar"/>
+  </library>
+</jdbcDriver>
+---
+
+The remaining configuration is provided by system properties.
+Add the following required properties to the `jvm.options` file to enable a new file based logger.
+
+[source, txt]
+----
+-Doracle.jdbc.Trace=true
+-DoracleLogFileName=<desired-log-name>.log
+-DoracleLogPackageName=<desired-package-to-trace>
+----
+
+Additional customization can be added to the `jvm.options` file. The following example lists the default values when none are configured.
```

Comment:
@dmuelle good question.  I wanted to show an example of other properties a user could configure AND the default values that will be used if they do not configure them in the same code block. 

If you think it is confusing I can go back can be more verbose.

Reply:
IF I'm understanding correctly, the example as configured now shows values you wouldn't actually need to configure this way, which could be confusing. If that's the sae, I think it would be better to list the default values separately (or point to where Oracle lists them) abnd give an example of how to supply alternative values. 

But if I'm making it more complicated than it actually is, then it might be fine as you have it :)